lib/item.rb

```diff
@@ -5,19 +5,21 @@ class Item
               :unit_price,
               :created_at,
               :updated_at,
-              :merchant_id
+              :merchant_id,
+              :repo
 
-  def initialize(item_info)
+  def initialize(item_info, repo)
     @id = item_info[:id].to_i
     @name = item_info[:name]
     @description = item_info[:description]
     @unit_price = item_info[:unit_price]
```

Comment:
I think the unit price needs a `.to_i` as well. This is worth doing before we merge, as it will allow all of the item repository tests to pass on everyone's machine (minus the 3 skipped tests).

Reply:
to_i or to_f? Do we need this to keep decimals? Not sure what the data looks like, those working in this class might know more than I on that.